cli/src/data.ts

```diff
@@ -23,7 +23,7 @@ async function simplefaceData (task: Task): Promise<data.DataSplit> {
 async function cifar10Data (cifar10: Task): Promise<data.DataSplit> {
   const dir = '../datasets/CIFAR10/'
   const files = (await fs.readdir(dir)).map((file) => path.join(dir, file))
-  const labels = Range(0, 24).map((label) => (label % 10).toString()).toArray()
+  const labels = Range(0, 24).map(_ => 'airplane').toArray() // TODO read labels in csv
```

Comment:
immutablejs has a specific way to have a repeated value, which is quite readable

```suggestion
  const labels = Repeat('airplane', 24).toArray() // TODO read labels in csv
```